Viewer/src/configuration/mappers.ts

```diff
@@ -52,6 +52,10 @@ class HTMLMapper implements IMapper {
                         val = true;
                     } else if (val === "false") {
                         val = false;
+                    } else if (val === "undefeind") {

```

Comment:
are you sure it should not be undefined ?

Reply:
yes I am. this is the mapper, which maps strings from the dom attributed to real objects

Reply:
so people will be able to set elements to be undefined (for example - templates.nav-bar.html="undefined")

Reply:
but undefeind and not undefined looks like a typo ?